app/src/layouts/common/sidebar/organizationsControl/organizationsControl.scss

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.organizations-control {
+  width: 240px;
+  height: 40px;
+  background-color: $COLOR--darkmode-gray-300;
+  border-radius: 8px;
+  padding: 10px 12px;
+  cursor: pointer;
+  margin-left: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.organization-btn {
+  height: 16px;
+  display: flex;
+  border: none;
+  padding: 0;
+  background: transparent;
+  color: $COLOR--darkmode-gray-100;
+  line-height: 16px;
+  font-size: 11px;
+  font-family: $FONT-ROBOTO-REGULAR;
+  cursor: pointer;
+}
+
+.arrow-icon {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.project-name {
+  color: $COLOR--darkmode-gray-100;
+  font-size: 13px;
+  font-family: $FONT-ROBOTO-MEDIUM;
+  line-height: 20px;
+  margin-top: 4px;
+}
+
+.open-popover {
+  width: 16px;
+  height: 16px;
+}
+
+.popover {
+  top: 102px !important;
+  background-color: $COLOR--darkmode-gray-400 !important;
```

Comment:
Let's agree to use `!important` directive only for overriding external libraries styles.
As this component managed by us, we can agree with design team to make another variant for it in the DS.
Or rework it to make more flexible.

Reply:
style={{ top, left }} Inline styles are difficult to override.

Reply:
So
> we can agree with design team to make another variant for it in the DS.
I suppose we need to add a new popover position variant, e.g. `right-top` that will set the correct properties for wapper and pseudo-elements if needed.

Reply:
Done.